railties/lib/rails/paths.rb

```diff
@@ -175,6 +175,12 @@ def to_ary
         @paths
       end
 
+      def expanded_paths
```

Comment:
Do you have any ideas how to deal with it without adding new methods to the class?